### PR TITLE
edits to install index

### PR DESCRIPTION
**Pull request by kalexand-rh**
This change is mostly edits for style.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
I made a couple of updates and posted a [preview build](http://file.rdu.redhat.com/kalexand/092018/edit_install_index/install/index.html).

**Comment by kalexand-rh** [comment]
@xltian, will you PTAL at this PR? It's mostly changes to improve the text, but I wanted to confirm that some of the material was still valid.

**Comment by xltian** [comment]
@jianlinliu from QE to review.

**Comment by kalexand-rh** [comment]
@jianlinliu will you PTAL?

**Comment by jianlinliu** [comment]
@gpei pls help have a check

**Comment by gpei** [comment]
In http://file.rdu.redhat.com/kalexand/092018/edit_install_index/install/index.html#containerized-required-images
Since only node system container will be used, so we also need to update this section.

Only node image is needed, and its name changed to :
openshift3/ose-node

File: install/index.adoc [hunk 1]
```diff
@@ -47,37 +38,39 @@ environment needs to be.
...
+High availability configurations improve fault tolerance. In this situation, you
+might use the xref:multi-masters-using-native-ha-colocated[Multiple Masters Using Native HA]
+example to set up your environment.
 
 * _Which installation type do you want to use: xref:planning-installation-types[RPM or system containers]?_
-Both installations provide a working {product-title} environment, but you might
-have a preference for a particular method of installing, managing, and updating
+If you use Red Hat Enterprise Linux (RHEL), you can use either method to install
+your {product-title} cluster, but if you use RHEL Atomic Host, you must use the
+system container method. You use the same method to install, manage, and update
```
[the remark this answers is not in the document]
Comment (sdodson) [reply install/index.adoc]: I think it just means that as far as admin interacting with the node service there is no difference between the two.

File: install/index.adoc [hunk 2]
```diff
@@ -47,37 +38,39 @@ environment needs to be.
...
-a basis for your environment.
+* _Does you need a xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#high-availability-masters[high availability] cluster?_
+High availability configurations improve fault tolerance. In this situation, you
+might use the xref:multi-masters-using-native-ha-colocated[Multiple Masters Using Native HA]
+example to set up your environment.
 
 * _Which installation type do you want to use: xref:planning-installation-types[RPM or system containers]?_
-Both installations provide a working {product-title} environment, but you might
-have a preference for a particular method of installing, managing, and updating
+If you use Red Hat Enterprise Linux (RHEL), you can use either method to install
```
[the remark this answers is not in the document]
Comment (sdodson) [reply install/index.adoc]: Right, there's no decision to be made with respect to RPM versus System Container. I think this entire bullet point and anything through the document that makes it look as if there's a decision to be made between RPM or System Container is removed.

File: install/index.adoc [hunk 3]
```diff
@@ -238,51 +230,53 @@ using the `native` HA method:
...
 xref:stand_alone_registry.adoc#install-config-installing-stand-alone-registry[Installing a Stand-alone Registry] for details on this scenario.
 
 [[planning-installation-types]]
-== Installation Types
+== Installation types for supported operating systems
+
+Starting in {product-title} 3.10, if you use RHEL Atomic Host, the system
+container method is used on that host. If you use RHEL
+as the underlying OS for a host, the RPM method is used by default, but you can
+choose to use the system container method to install
```
[the remark this answers is not in the document]
Comment (sdodson) [reply install/index.adoc]: Yeah, remove the option for choice.

File: install/index.adoc [hunk 4]
```diff
@@ -331,19 +325,19 @@ at the beginning of the installation process to ensure that these settings are
...
+=== systemd service names
 
 The installation process creates relevant *systemd* units which can be used to
 start, stop, and poll services using normal *systemctl* commands. For system
 container installations, these unit names match those of an RPM installation.
 
 [NOTE]
 ====
-The *etcd* package is slated to be removed from RHEL Atomic Host in the future.
+The *etcd* package will be removed from RHEL Atomic Host in the future.
```
Comment (sdodson) [inline install/index.adoc]: I'd remove this note, its no longer relevant in so much that now etcd runs as a pod on atomic host and the package was never removed and won't be prior to 4.0.

**Comment by kalexand-rh** [comment]
@sdodson, if I revert https://github.com/openshift/openshift-docs/pull/11994/commits/0f0822e541ffe5e3e0c706556d39da7151f39de9, will all the system container info be correct?

File: install/index.adoc [hunk 5]
```diff
@@ -240,63 +231,60 @@ using the `native` HA method:
...
 [[containerized-required-images]]
-=== Required Images for System Containers
+=== Required images for system containers
 
 The system container installation type makes use of the following images:
 
 ifdef::openshift-origin[]
 - *openshift/origin*
-- *openshift/node* (*node* + *openshift-sdn* + *openvswitch* RPM for client tools)
+- *openshift/ose-node* (*node* + *openshift-sdn* + *openvswitch* RPM for client tools)
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply install/index.adoc]: Oh, interesting, that doesn't match the current list at all.

@adellape, do these images sound right to you?

[the remark this answers is not in the document]
Comment (kalexand-rh) [reply install/index.adoc]: Scott says that it should just be openshift/origin-node

**Comment by kalexand-rh** [comment]
@gpei, I think that was the last issue. Does this change sound ok for 3.10 and 3.11?

**Comment by kalexand-rh** [comment]
Thank you! I've addressed all the comments and am going to merge.

**Pull request merged**